content/en/docs/concepts/security/application-security-checklist.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Application Security Checklist
+description: >
+  Baseline checklist for ensuring application security for Kubernetes Developers
+content_type: concept
+weight: 110
+---
+
+<!-- overview -->
+
+This checklist aims on providing basic guidelines on securing applications running in Kubernetes from a developer perspective. 
+This list is not meant to be exhaustive and is meant to evolve over time.
+
+<!-- The following is taken from the existing checklist created for Kubernetes admins. https://kubernetes.io/docs/concepts/security/security-checklist/ -->
+
+On how to read and use this document:
+
+- The order of topics does not reflect an order of priority.
+- Some checklist items are detailed in the paragraph below the list of each section.
+- This checklist assumes a `developer` is a Kubernetes cluster user that interacts with namespaced scope objects.
+
+
+{{< caution >}}
+Checklists are **not** sufficient for attaining a good security posture on their own. A good security posture requires constant attention and improvement, but a checklist can be the first step on the never-ending journey towards security preparedness. Some of the recommendations in this checklist may be too restrictive or too lax for your specific security needs. Since Kubernetes security is not "one size fits all", each category of checklist items should be evaluated on its merits.
+{{< /caution >}}
+
+
+<!-- body -->
+## Base security hardening
+The following checklist provide a base security hardening recommendations that would apply to most applications deploying to Kubernetes.
+
+### Application design
+- [ ] Following the right [design principles for the application.](https://kubernetes.io/blog/2018/03/principles-of-container-app-design/)
```

Comment:
I don't recommend linking to that article. Find a better source, such as https://www.cncf.io/wp-content/uploads/2022/06/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf